KevS-SK1552-Update-Glue-records-for-DomainDNS (#6398)
ovh/docs: pages/web_cloud/domains/glue_registry/guide.fr-fr.md

### Étape 2 : ajouter les enregistrements « GLUE » <a name="step2"></a>

> > [!alert]

Author: ```suggestion
> [!warning]
```

>

> > Pour les extensions *.eu*,*.it*, *.be* et *.de*, les registres de ces extensions ne considèrenet pas les enregistrements « GLUE » comme des « objets » mais comme des « attributs ».

Reviewer: ```suggestion
> Les registres des extensions *.eu*,*.it*, *.be* et *.de* ne considèrent pas les enregistrements « GLUE » comme des « objets » mais comme des « attributs ».
```

>

> > Par conséquent et pour ces extensions, passez **directement à l'étape 3** du présent guide sans réaliser l'étape 2.

Reviewer: ```suggestion
> Par conséquent, pour ces extensions, passez **directement à l'étape 3** de ce guide sans réaliser l'étape 2.
```

 
Positionnez-vous sur l'onglet `Serveurs DNS`{.action} puis cliquez sur `Modifier les serveurs DNS`{.action}. Remplacez alors vos serveurs DNS actuels par ceux que vous souhaitez utiliser en tant que serveur DNS personnalisés. 

> > [!alert]

Author: ```suggestion
> [!warning]
```

>

> > Si vos serveurs DNS personnalisées ont été créées avec les extensions *.eu*,*.it*, *.be* et/ou *.de*, renseignez **obligatoirement** l'adresse IP asociée respectivement à chacun de vos serveurs DNS personnalisés.

Reviewer: ```suggestion
> Si vos serveurs DNS personnalisés ont été créés avec les extensions *.eu*,*.it*, *.be* ou *.de*, renseignez **obligatoirement** l'adresse IP associée respectivement pour chacun de vos serveurs DNS personnalisés.
```

>

> > Le cas échéant, les serveurs DNS personnalisés ne seront pas pris en compte correctmeent et ne fonctionneront donc pas avec votre nom de domaine.

Reviewer: ```suggestion
> Sans cela, les serveurs DNS personnalisés ne seront pas pris en compte correctement et ne fonctionneront donc pas avec votre nom de domaine.
```
